docs/src/guide/parallelization.md

```diff
@@ -32,10 +31,8 @@ inside this datastructure, any timing measurement should first reset
 this timer before running the calculation of interest.
 
 For example to measure the timing of an SCF:
-```@example parallelisation
-import TimerOutputs
-
-TimerOutputs.reset_timer!(DFTK.timer)
+```@example parallelization
+DFTK.reset_timer!(DFTK.timer)
```

Comment:
This works? Takes me by surprise.

Reply:
Yup. I guess because DFTK uses TimerOutputs, which exports it, so reset_timer! is "known" in the DFTK namespace

Reply:
Yes indeed, but for some reason I never thought about that.